docs/BaseSchema.md

```diff
@@ -0,0 +1,174 @@
+# BaseSchema
```

Comment:
Are these tables and stored procedures something a DBA would have to make manually the first time they upgrade a server, or do we have a tool to make them?

Reply:
Good question! As mentioned in the doc, for now DBA would need to execute it manually if it doesn't exist https://github.com/microsoft/fhir-server/pull/1120/files#diff-88920ff7161e7d92d1ae16e8c15c646fR7

But we had thought about it and might create a backlog item to execute the BaseSchema by the tool itself.